docs/src/pages/_document.page.tsx

```diff
@@ -25,16 +25,20 @@ const getCSPContent = (context: Readonly<HtmlProps>) => {
       style-src 'self' 'unsafe-inline';
       font-src 'self' data:;
       frame-src *.codesandbox.io;
-      script-src 'unsafe-eval' 'self' '${cspInlineScriptHash}'
+      img-src 'self' cm.everesttech.net amazonwebservices.d2.sc.omtrdc.net;
```

Comment:
Do you want to add `dpm.demdex.net`; here as well?
```suggestion
      img-src 'self' cm.everesttech.net amazonwebservices.d2.sc.omtrdc.net dpm.demdex.net;
```